packages/typo3-docs-theme/assets/sass/_component_fieldlist.scss

```diff
@@ -25,6 +7,7 @@
         padding-left: 0;
         padding-right: .5em;
         margin-bottom: $spacer * .25;
+        hyphens: manual;
```

Comment:
"manual" is the default value, so no need to define it:
https://developer.mozilla.org/en-US/docs/Web/CSS/hyphens?retiredLocale=de#values

Reply:
removed it